src/Settings/Identity/Banner.jsx

```diff
@@ -0,0 +1,72 @@
+let { open, onClick } = props;
+
+const idOSLearnLink = "https://idos-1.gitbook.io/idos-docs";
+
+const Wrapper = styled.div`
+  display: ${(p) => (p.open ? "flex" : "none")};
+  align-items: start;
+  gap: 30px;
+  user-select: none;
+  animation-duration: 400ms;
+  animation-timing-function: cubic-bezier(0.16, 1, 0.3, 1);
+  will-change: transform, opacity;
+  animation-name: slideUpAndFade;
+
+  @keyframes slideUpAndFade {
+    from {
+      opacity: 0;
+      transform: translateY(2px);
+    }
+    to {
+      opacity: 1;
+      transform: translateY(0);
+    }
+  }
+`;
+
+const Card = styled.div`
+  display: flex;
+  align-items: start;
+  gap: 14px;
+  padding: 16px 26px;
+  border-radius: 12px;
+  background: #ededed;
+  margin-bottom: 24px;
+`;
+
+const Icon = styled.i`
+  color: #697177;
+  font-size: 20px;
+  margin-top: 2px;
+
+  [type="button"] {
+    cursor: pointer;
+  }
+`;
+
+const Text = styled.span`
+  color: #1d1412;
+`;
+
+const TextLink = styled.a`
+  color: inherit;
+  text-decoration: underline;
+`;
+
+return (
+  <Wrapper open={open}>
+    <Card>
+      <Icon className="ph ph-info" />
+      <Text>
+        To ensure the privacy of your data, all credentials and preferences you
+        enter on this settings page will be securely stored in a decentralized
+        identity operating system (idOS). Learn more about idOS{" "}
+        <TextLink href={idOSLearnLink} target="_blank">
+          here
+        </TextLink>
+        .
+      </Text>
+      <Icon className="ph ph-x" type="button" onClick={onClick} />
```

Comment:
@xuan-ytr  @shelegdmitriy I noticed that this requires a click, should this display on hover? I thought we have a pattern of tools tips displaying on hover instead of requiring a click.

Reply:
But according to the design (or maybe I understand this in wrong way) this icon relates to this banner
<img width="679" alt="Знімок екрана 2023-11-03 о 22 00 23" src="https://github.com/near/near-discovery-components/assets/34593263/550950b9-47ad-4943-8dde-3a907c76df53">
That's why I thought that icon should be clickable